net/ping/ping.go

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package ping provides utilities for ICMP pings. It currently delegates to
+// ping programs, in order to avoid capability and API restrictions.
+package ping
+
+import (
+	"errors"
+	"io"
+	"os"
+	"os/exec"
+	"regexp"
```

Comment:
we've historically not used used the regexp package anywhere reachable from iOS because it brings in huge unicode tables.

Reply:
Ah, that's a very fair point, and this is relatively easy to parse by hand, I'll do so.